.github/CODEOWNERS

```diff
@@ -106,8 +106,10 @@
 
 /templates/**/copilot-plugin-from-scratch @hund030 @eriolchan @huimiu
 /templates/**/dashboard-tab @hund030 @eriolchan @huimiu
-/templates/**/non-sso-tab @hund030 @eriolchan @huimiu @JerryYangKai
-/templates/**/sso-tab @JerryYangKai @eriolchan @hund030 @huimiu
+/templates/**/non-sso-tab @hund030 @eriolchan @JerryYangKai
```

Comment:
You can also update Kai to Yimin for non-ssr templates

Reply:
Updated.